packages/atomic/src/components/facets/atomic-segmented-facet/atomic-segmented-facet.pcss

```diff
@@ -0,0 +1,16 @@
+@import '../../../global/global.pcss';
+@import '../facet-common.pcss';
+@import '../facet-search/facet-search.pcss';
```

Comment:
Since there is no search for segmented facet, we don't need to import that additional css